composer.json

```diff
@@ -6,11 +6,11 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.2 || ^8.0",
+        "php": "^7.2 || ^8.0 || ^8.1",
```

Comment:
`|| ^8.1` should not be required as it is covered by `^8.0` isn't it?

Reply:
Yes. It is enough